CMakeLists.txt

```diff
@@ -79,13 +79,13 @@ endif()
 # ============
 
 set(xtl_REQUIRED_VERSION 0.7.0)
-set(xeus_REQUIRED_VERSION 3.0.5)
+set(xeus_REQUIRED_VERSION 3.2.0)
 
 if (NOT TARGET xtl)
     find_package(xtl ${xtl_REQUIRED_VERSION} REQUIRED)
 endif ()
 if (NOT TARGET xeus AND NOT TARGET xeus-static)
-    find_package(xeus ${xeus_REQUIRED_VERSION} REQUIRED)
```

Comment:
We should have a cmake range here. We should be able to build correctly xeus-cpp outside of the conda context. In that case we will need cmake to define the supported ranges, too.

Reply:
Hey Vassil, not fully sure what you mean by a range here. Something like this ?
```
set(xeus_MINIMUM_VERSION 3.2.0)
set(xeus_MAXIMUM_VERSION 4.0.0)

if (NOT TARGET xtl)
    find_package(xtl ${xtl_REQUIRED_VERSION} REQUIRED)
endif ()
if (NOT TARGET xeus AND NOT TARGET xeus-static)
    find_package(xeus ${xeus_MINIMUM_VERSION} REQUIRED)
    if (xeus_VERSION VERSION_LESS xeus_MINIMUM_VERSION OR xeus_VERSION VERSION_GREATER xeus_MAXIMUM_VERSION)
        message(FATAL_ERROR "xeus version must be between ${xeus_MINIMUM_VERSION} and ${xeus_MAXIMUM_VERSION}")
    endif ()
endif ()
```

Reply:
`find_package(xeus ${xeus_REQUIRED_VERSION} REQUIRED)` should probably stay because that tells the cmake that the version in `xeus_REQUIRED_VERSION` is the minimal required version.

Reply:
Yeah I think that makes sense.